themes/red-theme/lms/templates/header.html

```diff
@@ -1,6 +1,6 @@
 ## mako
 <%namespace name='static' file='static_content.html'/>
-<%namespace file='main.html' import="login_query, stanford_theme_enabled"/>
```

Comment:
@mattdrayer I looked at all the usages of `stanford_theme_enabled` in `edx-platform` and `edx-microsites` and at all places file is either inherited from `main.html` or `stanford_theme_enabled` is imported from `main.html`. So, I think we can safely remove unused import from here.

`course_about.html` is inherited from `main.html` ([here](https://github.com/edx/edx-platform/blob/master/lms/templates/courseware/course_about.html#L11)) so `self.stanford_theme_enabled` will be resolved via inheritance chain.

Please let me know if I am missing something.